caffe2/CMakeLists.txt

```diff
@@ -955,6 +955,7 @@ endif()
 if(USE_ROCM)
   set(CUDA_LINK_LIBRARIES_KEYWORD PRIVATE)
   hip_add_library(torch_hip ${Caffe2_HIP_SRCS})
+  target_link_libraries(torch_hip PRIVATE __caffe2_oort)
```

Comment:
Shouldn't this be guarded by `if(USE_FLASH_ATTENTION)`?

Reply:
@malfet After trying for a few more cycles, no, we can't, unfortunately.

`USE_FLASH_ATTENTION` and `USE_MEM_EFF_ATTENTION` are used in `cmake/Dependencies.cmake` to enable FA/MEFF, but these two flags depends on `USE_ROCM`, which defaults to True and will be set correctly inside `cmake/Dependencies.cmake`.